docs/install.md

```diff
@@ -5,7 +5,7 @@ This project depends on LLVM and Clang. lsif-clang itself should be built agains
 ### Ubuntu (20.04)
 
 ```sh
-apt install llvm-10 clang clang-10 libclang-10-dev cmake
+apt install llvm-10 clang clang-10 libclang-10-dev cmake binutils-dev
```

Comment:
I'm wondering why this part makes sense. Later you're using:

```
BACKWARD_HAS_DWARF
```

According to https://github.com/bombela/backward-cpp/blob/master/backward.cpp#L14-L34, `BACKWARD_HAS_DWARF` corresponds to `libdwarf-dev`, whereas `binutils-dev` requires `BACKWARD_HAS_BFD`.